app/frontend/src/components/calendar/Calendar.js

```diff
@@ -1,20 +1,28 @@
-import React from 'react'
+import { useContext, useEffect, useState } from "react";
 import FullCalendar from '@fullcalendar/react'
 import dayGridPlugin from '@fullcalendar/daygrid'
 import timeGridPlugin from '@fullcalendar/timegrid'
 import listPlugin from '@fullcalendar/list'
 import interactionPlugin from '@fullcalendar/interaction'
+import iCalendarPlugin from '@fullcalendar/icalendar'
 import bootstrap5Plugin from '@fullcalendar/bootstrap5';
 import Container from 'react-bootstrap/esm/Container';
 
 import {CalendarHttpClientContext} from '../../App';
 
 function Calendar() {
+  const calendarHttpClient = useContext(CalendarHttpClientContext)
+  const [events, setEvents] = useState(null);
+  useEffect(() => {
+    setEvents(null);
+    calendarHttpClient.getEventsFromRapla('https://rapla.dhbw-karlsruhe.de/rapla?page=ical&user=eisenbiegler&file=TINF21B4').then((ev) => {setEvents(ev); console.log(ev);});
+  }, []
+  );
   return (
     <Container>
       <h1>Calendar</h1>
       <FullCalendar
-        plugins={[ bootstrap5Plugin, dayGridPlugin, listPlugin, timeGridPlugin ]}
+        plugins={[ bootstrap5Plugin, dayGridPlugin, listPlugin, timeGridPlugin, iCalendarPlugin ]}
```

Comment:
iCalendarPlugin is not in the package.json -> removed from import -> not usable here, remove